orchestrator/db/sorting/subscription.py

```diff
@@ -0,0 +1,20 @@
+from orchestrator.db import SubscriptionTable
+from orchestrator.db.sorting.sorting import generic_sort
+
+VALID_SORT_KEY_LIST = [
+    "subscription_id",
+    "product_id",
+    "customer_id",
```

Comment:
customer_id?

Also, is this list different from just all the fields in a subscription?

Reply:
pretty much all subscription fields, tried just taking all the fields from the table, but then you will have to filter all the extra functions and vars, so thought explicit would be more appropriate.

Reply:
does need to change to something similar to filtering, since you want to be able to sort on other tables

Reply:
will be done in a new PR